packages/openapi3/src/cli/actions/convert/transforms/transforms.ts

```diff
@@ -80,6 +127,64 @@ function scanOperationForMultipartSchemas(
   }
 }
 
+function scanPathForSSESchemas(
+  path: OpenAPI3PathItem | OpenAPIPathItem3_2,
+  context: Context,
+): void {
+  if (!("get" in path && path.get && path.get.responses)) {
```

Comment:
this is wrong, a streaming response could come from any operation that MAY return a response body